molo/surveys/utils.py

```diff
@@ -44,9 +44,13 @@ def __init__(self, object_list, data=dict(), answered=dict()):
 
         # add the missing data
         self.new_answers.update({
-            checkbox.clean_name: 'off'
+            checkbox.pk_clean_name: 'off'
             for checkbox in self.missing_checkboxes
         })
+        self.new_answers.update({
+            checkbox.clean_name: 'off'
+            for checkbox in self.pk_cleaned_missing_checkboxes
+        })
```

Comment:
@moh-moola Doesn't the above code mean that we're adding the missing checkboxes twice? Once with their clean_name and once with their pk_clean_name
And how does this affect checkboxes that were answered? because wouldn't that exist with their clean_name but not with their pk_clean_name and so we'll add conflicting information? Or will they get added before this?